src/parser/src/BinaryExprNode.cpp

```diff
@@ -0,0 +1,10 @@
+#include "BinaryExprNode.hpp"
+
```

Comment:
Another undetected rename

Reply:
This is renamed fine? The old file has been deleted so it probably has some trouble finding the original.